TLM/TLM/UI/SubTools/TTL/TimedTrafficLightsTool.cs

```diff
@@ -136,11 +136,16 @@ public TimedTrafficLightsTool(TrafficManagerTool mainTool)
             if (HoveredNodeId <= 0 || nodeSelectionLocked || !Flags.MayHaveTrafficLight(HoveredNodeId)) {
                 return;
             }
-            bool ctrlDown = Input.GetKey(KeyCode.LeftControl) || Input.GetKey(KeyCode.RightControl);
-            if (ctrlDown) {
-                AutoTimedTrafficLights.ErrorResult res = AutoTimedTrafficLights.Setup(HoveredNodeId);
+            if (Shortcuts.ControlIsPressed) {
+                AutoTimedTrafficLights.ErrorResult res;
+                if (Shortcuts.AltIsPressed) {
+                    res = AutoTimedTrafficLights.SetupSingleFar(HoveredNodeId);
+                } else {
+                    res = AutoTimedTrafficLights.Setup(HoveredNodeId);
+                }
                 string message = null;
 
+                Log._Debug(res.ToString());
```

Comment:
Either log this with explanation useful for support and log reading, or remove debug log